### ISSUE TITLE
Update the server blurb

### ISSUE THREAD

**Issue by jmacdotorg** [Five-Borough-Fedi-Project/Five-Borough-Fedi-Project.github.io#8]
I'd like to update the server blurb to reflect our growth over the last two years while retaining its fun vibe—while also fitting within the character limit implied by https://joinmastodon.org/servers.

Current text:
> Regional New York City 🐘 🗽 instance! A place to engage with your local neighbors and friends

Proposed update:
> 🗽 A regional server for New York City! Engage with your neighbors, friends, and the whole fediverse. Operated with ❤️ by a dedicated nonprofit.

* Updates to use the "server" terminology that Mastodon favors these days
* Adds a nod to our nonprofit status while making an "I❤️NY" reference

WYT @seano-vs

**Comment by seano-vs** [comment]
Proposal: Let's drop the third sentence. This blurb's primary function (in my opinion) is to pitch our server on https://joinmastodon.org/servers and if you look at the servers on there, this text would be one of the largest by character count on that page (I'm thinking about glance value for when people scroll there).

how about:

> 🗽 A regional non-profit server for New York City! Engage with your neighbors, friends, and the whole fediverse.

That's just my two cents, if you feel strongly about the original text I'm ok with it

**Comment by jmacdotorg** [comment]
Heh, I feel a little bit pushy about this one!

> This blurb's primary function (in my opinion) is to pitch our server on https://joinmastodon.org/servers

Quite right! And the "Branding" form on the Mastodon admin UI prompts it this way:

> A short description to help uniquely identify your server. Who is running it, who is it for?

My intent is to answer both halves of that question, with succinctness and spirit.

> If you look at the servers on there, this text would be one of the largest by character count on that page

I'm not so sure about that! That page is hardcore about character limits and lots of servers overrun it; there's quite a few trailing ellipses. While many servers are super-short, none are particularly long (because the page doesn't allow it), and I intentionally wrote this to land shy of the cutoff.

Tell you what... since this is easy to change on the fly, lemme put up my version and then let's see how it looks. If we agree that it still looks too wordy we'll pop it down to your suggested edit. SG?

**Comment by jmacdotorg** [comment]
Here's what it currently looks like on the servers list, surrounded by its nearest neighbors:

<img width="796" alt="Screenshot 2024-11-02 at 3 21 04 PM" src="https://github.com/user-attachments/assets/1dcadb98-aa47-4410-844d-5561f8570f9b">
